src/pixelator/collapse/process.py

```diff
@@ -463,9 +464,9 @@ def create_edgelist(
         - `upib`, the upib of the fragment
         - `umi`, the umi of the fragment
         - `count`, the number of upib's associated with the fragment
-        - `umi_unique_count`, the number of unique molecules (based on upia+umi)
+        - `unique_fragment_count`, the number of unique molecules (based on upia+umi)
```

Comment:
I have renamed this to `unique_molecules_count`. Something to keep in mind is that this column is calculated to have some idea of how many error corrected molecules are represented into a single "fixed" molecule but it is not written to the edgelist file anymore. We still need to calculate this column because we have a `collapsed_molecule_count_stats` `SummaryStatistic` object in the collapse.report.json.

This is because https://github.com/PixelgenTechnologies/pixelator/pull/67/files has removed the `umi_unique_count` column (ie the precursor of `unique_molecules_count`)